crates/holochain_cascade/src/authority/get_element_query.rs

```diff
@@ -89,8 +89,6 @@ impl Query for GetElementOpsQuery {
             DhtOpType::StoreElement => {
                 if state.header.is_none() {
                     state.header = Some(dht_op.map(|d| d.header));
-                } else {
```

Comment:
This can't happen as the hash of a dht op is derived from the header so there literally can't be multiple in the database.